modules/jt-comparing-jenkins-tekton-concepts.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-comparing-jenkins-tekton-concepts_{context}"]
+= Comparing Jenkins and Tekton concepts
+
+toc::[]
+
+
+This section summarizes the basic terms used in Jenkins and Tekton, and compares the equivalent terms.
+
+== Jenkins terminology
+Jenkins offers declarative and scripted pipelines that are extensible using shared libraries and plugins. Some basic terms in Jenkins are as follows:
+
+* *Pipeline*: Automates the entire process of building, testing, and deploying applications, using the link:https://groovy-lang.org/[Groovy] syntax.
+* *Node*: A machine capable of orchestrating and/or executing a scripted pipeline.
```

Comment:
IBM SG, page 25. The SG doesn't want us to use `and/or` Consider rephrasing to: `A machine capable of either orchestrating or executing a scripted pipeline.` 

This rephrase tells the user that the machine can do both, but the user has to choose for it to either orchestrate or execute (it can't do both simultaneously).